tests/ballerina-test/src/test/java/org/ballerinalang/test/documentation/DocumentationTest.java

```diff
@@ -127,7 +127,7 @@ public void testDocStruct() {
     @Test(description = "Test doc function.")
     public void testDocFunction() {
         CompileResult compileResult = BCompileUtil.compile("test-src/documentation/function.bal");
-        Assert.assertEquals(0, compileResult.getWarnCount());
+        Assert.assertEquals(1, compileResult.getWarnCount());
```

Comment:
Why you have changed the assertion in this test, this means you have ignored the error?